### DISCUSSION TITLE
TextToSpeech  rate and current reading index

### DISCUSSION THREAD

**Discussion by gogo1008** [dotnet/maui#19315] [Q&A]
### Description

Does TextToSpeech have a speech rate and return the index of the current reading, if it has been implemented, please provide the use case or documentation

### Steps to Reproduce

TextToSpeech.Default.SpeakAsync(timeLeft)

### Link to public reproduction project repository

https://github.com/dotnet/maui

### Version with bug

7.0.49

### Last version that worked well

7.0.49

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

Andriod IOS Window

### Did you find any workaround?

Does TextToSpeech have a speech rate and return the index of the current reading, if it has been implemented, please provide the use case or documentation

### Relevant log output

```shell
Does TextToSpeech have a speech rate and return the index of the current reading, if it has been implemented, please provide the use case or documentation
```

**Comment by samhouts** [comment]
Documentation is available at https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/device-media/text-to-speech?view=net-maui-8.0&tabs=android. I'll convert this to a discussion to see if others have ideas on how to get waht you want. Thanks!
